OvmfPkg/OvmfPkgX64.dsc

```diff
@@ -848,7 +926,7 @@
 !ifdef $(CSM_ENABLE)
   OvmfPkg/Csm/BiosThunk/VideoDxe/VideoDxe.inf {
     <LibraryClasses>
-      PcdLib|MdePkg/Library/DxePcdLib/DxePcdLib.inf
+     PcdLib|MdePkg/Library/DxePcdLib/DxePcdLib.inf

```

Comment:
Useless change, please revert.

Reply:
No... I didn't ask to remove the line! You were supposed to revert the change introduced in this line (i.e. the unnecessary indentation change).  Thankfully you did it.